app/controllers/idv/in_person/verify_info_controller.rb

```diff
@@ -0,0 +1,176 @@
+        idv_session.vendor_phone_confirmation = false
+        idv_session.user_phone_confirmation = false
+
+        Idv::Agent.new(pii).proof_resolution(
+          document_capture_session,
+          should_proof_state_id: should_use_aamva?(pii),
+          trace_id: amzn_trace_id,
+          user_id: current_user.id,
+          threatmetrix_session_id: flow_session[:threatmetrix_session_id],
+          request_ip: request.remote_ip,
+        )
+
+        redirect_to idv_in_person_verify_info_url
+      end
+
+      private
+
+      def next_url
+        idv_phone_url
+      end
+
+      def prev_url
+        idv_in_person_url
+      end
+
+      def renders_404_if_flag_not_set
+        render_not_found unless IdentityConfig.store.in_person_verify_info_controller_enabled
+      end
+
+      def add_proofing_component
+        ProofingComponent.
+          create_or_find_by(user: current_user).
+          update(document_check: Idp::Constants::Vendors::USPS)
+      end
+
+      # copied from address_controller
+      def confirm_ssn_step_complete
+        return if pii.present? && pii[:ssn].present?
+        redirect_to idv_in_person_url
+      end
+
+      def confirm_profile_not_already_confirmed
+        # todo: should this instead be like so?
+        # return unless idv_session.resolution_successful == true
+        return unless idv_session.profile_confirmation == true
+        redirect_to idv_phone_url
+      end
+
+      def pii
+        @pii = flow_session[:pii_from_user] if flow_session
+      end
+
+      def delete_pii
+        flow_session.delete(:pii_from_doc)
+        flow_session.delete(:pii_from_user)
+      end
+
+      def current_flow_step_counts
+        user_session['idv/in_person_flow_step_counts'] ||= {}
+        user_session['idv/in_person_flow_step_counts'].default = 0
+        user_session['idv/in_person_flow_step_counts']
+      end
+
+      def increment_step_counts
+        current_flow_step_counts['verify'] += 1
+      end
+
+      # override StepUtilitiesConcern
+      def flow_session
+        user_session['idv/in_person']
+      end
+
+      def analytics_arguments
+        {
+          flow_path: flow_path,
+          step: 'verify',
+          step_count: current_flow_step_counts['verify'],
+          analytics_id: 'In Person Proofing',
+          irs_reproofing: irs_reproofing?,
+        }.merge(**acuant_sdk_ab_test_analytics_args)
+      end
+
+      # copied from verify_base_step. May want reconciliation with phone_step
+      def process_async_state(current_async_state)
+        if current_async_state.none?
+          idv_session.resolution_successful = false
+          render 'idv/verify_info/show'
+        elsif current_async_state.in_progress?
+          render 'shared/wait'
+        elsif current_async_state.missing?
+          analytics.idv_proofing_resolution_result_missing
+          flash.now[:error] = I18n.t('idv.failure.timeout')
+          render 'idv/verify_info/show'
```

Comment:
The pattern we seem to be using a view that renders a shared template. I think that might be _an_ option here, but let's get some attention from @tomas-nava and @sheldon-b.

If we are going to change the way the verify step appears in in-person it may be worth having entirely different views without a shared template. That would have an added benefit of allowing us to remove a handful of ivars from `#new` above.